tests/integration/targets/pacman/tasks/package_urls.yml

```
@@ -19,7 +23,14 @@
     - name: Get URL for {{url_pkg}}
       command:
         cmd: pacman --sync --print-format "%l" {{url_pkg}}
-      register: url_pkg_url
+      register: url_pkg_stdout
+    - name: Download {{url_pkg}} pkg
+      get_url:
+        url: '{{url_pkg_stdout.stdout}}'
+        dest: '{{url_pkg_path}}'
```

I think we also need the `.sig` file so pacman doesn't complain.
```suggestion
        dest: '{{url_pkg_path}}'
    - name: Download {{url_pkg}} pkg sig
      get_url:
        url: '{{url_pkg_stdout.stdout}}.sig'
        dest: '{{url_pkg_path}}.sig'
```